### PR TITLE
Integrate JSCS (and run against src/hello.js initially)

### PR DESCRIPTION
**Pull request by WebSeed**
Ready for review.

### CODE REVIEW

**Comment by MrSwitch** [comment]
@WebSeed i realize this is gonna come as a huge shock,... but, i'm a tabs fan.

**Comment by WebSeed** [comment]
@MrSwitch that's why I opened the PR early :smile: I will switch to tabs. Don't worry, I won't be merging any of this - you hit the merge if / when you're happy. I just think it's useful to have a convention (whatever that may be) that everyone can adhere to.

File: .jscsrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "preset": "airbnb",
+  "requireCamelCaseOrUpperCaseIdentifiers": "ignoreProperties",
+  "validateIndentation": "\t"
```
Comment (WebSeed) [inline .jscsrc]: Use tabs.

File: Gruntfile.js [hunk 2]
```diff
@@ -1,70 +1,64 @@
...
-	grunt.initConfig({
-		pkg: grunt.file.readJSON('package.json'),
-		jshint: {
-			files: ['Gruntfile.js', 'src/**/*.js', 'tests/**/*.js'],
-			options: {
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    // Code formatting / linting
+    jscs: {
+      src: 'src/hello.js',
```
Comment (WebSeed) [inline Gruntfile.js]: Applying to one file for now - will add the others (including tests) if/when happy with changes.

**Comment by WebSeed** [comment]
Hi @MrSwitch - the JSCS tests are now passing for one file `src/hello.js` (using tabs). It required quite a lot of changes. I don't think this has resulted in any news bugs (Mocha tests pass) but worth a quick look. Merge if you're happy and I'll start looking at doing this for other files.

File: Gruntfile.js [hunk 3]
```diff
@@ -1,70 +1,64 @@
 module.exports = function(grunt) {
 
-	grunt.initConfig({
-		pkg: grunt.file.readJSON('package.json'),
-		jshint: {
```
Comment (MrSwitch) [inline Gruntfile.js]: I'm not sure JSHint has been made redundant by JSCS. I re-enabled it and it highlighted some issues. What do you think?

Comment (WebSeed) [inline Gruntfile.js]: Yes, I was thinking the same thing - certainly no harm in them both running anyway.

File: Gruntfile.js (hunk 2 above)
Comment (MrSwitch) [inline Gruntfile.js]: Sounds great! If you don't mind returning the Gruntfile back to tabs I think we can merge the changes thus. Or if you'd prefer it to be in one PR, up to you.

File: Gruntfile.js [hunk 4]
```diff
@@ -5,38 +5,34 @@ module.exports = function(grunt) {
...
-				// options here to override JSHint defaults
 				globals: {
 					console: true,
 					module: true,
 					document: true
 				},
-				// don't check dot notation
 				sub: true
 			}
 		},
```
Comment (WebSeed) [inline Gruntfile.js]: Re-enabled JSHint and fixed issues identified.

**Comment by WebSeed** [comment]
@MrSwitch - Converted Gruntfile.js back to tabs (and added to JSCS test).

File: Gruntfile.js (hunk 4 above)
Comment (MrSwitch) [inline Gruntfile.js]: Thanks

**Pull request merged**